pkg/k8s/synced/crd.go

```diff
@@ -48,32 +48,29 @@ const (
 )
 
 var (
-	crdResourceNames = []string{
+	AgentCRDResourceNames = []string{
```

Comment:
Would be good to have a simple godoc on this explaining what this list is and why it contains what it does.

Reply:
added